src/dropdown/dropdown.scss

```diff
@@ -213,23 +213,59 @@ $height-small: 30px !default;
   }
 
   &.Select--multi {
+    height: auto;
+
     &.has-value .Select-input {
       position: relative;
     }
 
+    &.has-value .Select-control {
+      padding: 0;
+    }
+
+    .Select-placeholder {
+      padding: 0 10px;
+    }
+
+    .Select-control {
+      padding: 0;
+      display: table;
+      border-spacing: 0;
+      border-collapse: separate;
+      outline: none;
+      overflow: hidden;
+      position: relative;
+      width: 100%;
+    }
+
+    .Select-multi-value-wrapper,
+    .Select-arrow-zone,
+    .Select-clear-zone {
+      display: table-cell;
+      position: relative;
+      vertical-align: middle;
+    }
+
+    .Select-multi-value-wrapper {
+      width: auto;
+    }
+
     .Select-control .Select-value {
       margin-right: 5px;
+      margin-left: 5px;
```

Comment:
here you're adding an additional `5px` margin between every value. I supposed your intent was to add space from left-size of the dropdown. Maybe adding a padding where needed is a better solution.